### ISSUE TITLE
2.9.0-beta2 Pre-release Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3112]
- #3083
- #3078 
- #3085 
- #3100 
- #3088 
- #2759 
- #3106 
- #3107 
- #3105 
- #3095
- #3093
- #3094 

## Pick list
```
git cherry-pick 5060759e222eb2a7587edecc5f8674c7007086fe
git cherry-pick 5c387920f4633126a0df66c7e49a6dd953f423b0
git cherry-pick 30a2535a16af43869c35f5de82d0f2671dff7489
git cherry-pick 0202746413a40837891d01abef766a2b151a2a19
git cherry-pick 81fe8413db56776573d3e28e4175f4595018cd1e
git cherry-pick 43489ec92de904167383e0e7de7ab2b284e34bc6
git cherry-pick bd22af7dfb68dba7e3d34634ddbd98382656d3a1
git cherry-pick fbdb3d37c0000c42a4d119dd3cf22d0699cab9df
git cherry-pick 132c5d5f138b26012a2be64b07ca194793866fc0
git cherry-pick 43490b9d3054ee0125ba6136fc191942f30f90ae
git cherry-pick a0f3d8f7c53c7768e952796a6fbdcbd3c6878f1a
git cherry-pick 1eb77ea3110cfba5e8daea944693b97a01c43f83
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.9.0-beta1..HEAD --pretty='format:- %s'`
```
_Note: This release contains a security update for Drupal core that addresses a moderately critical security vulnerability ([SA-CORE-2024-001](https://www.drupal.org/SA-CORE-2024-001)).  Only sites that have the core comment module installed are affected by this vulnerability._

- Update drupal/ckeditor_bs_grid requirement from 2.0.10 to 2.0.11 (#3094)
- Update drupal/migrate_plus requirement from 6.0.1 to 6.0.2 (#3093)
- Update drupal/paragraphs requirement from 1.16.0 to 1.17.0 (#3095)
- Close #3104 fix typo in az_news_feeds schema. (#3105)
- Closes #3102 Some CKEditor editable tags are removed and can't be restored (#3107)
- Closes #3097 Changes to rendered HTML after update to Drupal 10.2 (#3106)
- Closes #2758: Add patch to address Easy Breadcrumb deprecation warnings. (#2759)
- Update drupal/quick_node_clone requirement from 1.16.0 to 1.17.0 (#3088)
- #3099 Update Drupal core to 10.2.2 for SA-CORE-2024-001 (#3100)
- Closes #3084 Event edit page displays User deprecated function error (#3085)
- Closes #3077: Address PHP 8.1+ deprecation warnings caused by card links with empty URLs. (#3078)
- Closes #3082 Vertical tabs in Media Library modal are missing styling (#3083)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.9.0-beta2
